dialogue-core/src/main/java/com/palantir/dialogue/core/ConcurrencyLimitedChannel.java

```diff
@@ -42,22 +42,24 @@
  */
 final class ConcurrencyLimitedChannel implements LimitedChannel {
     private static final Void NO_CONTEXT = null;
+    private static final Supplier<Long> SYSTEM_NANOTIME = System::nanoTime;
 
     private final Channel delegate;
-    private final LoadingCache<Endpoint, Limiter<Void>> limiters;
+    private final LoadingCache<Endpoint, SimpleLimiter<Void>> limiters;
 
     @VisibleForTesting
-    ConcurrencyLimitedChannel(Channel delegate, Supplier<Limiter<Void>> limiterSupplier) {
+    ConcurrencyLimitedChannel(Channel delegate, Supplier<SimpleLimiter<Void>> limiterSupplier) {
```

Comment:
maybe split these changes to a separate PR?

Reply:
This was just to allow me to log the permits - have reverted as I've deleted the corresponding logging